### ISSUE TITLE
Conjure-up kubernetes juju bootstrap fail

### ISSUE THREAD

**Issue by lavoiee** [conjure-up/conjure-up#1250]
## Report

Thank you for trying **conjure-up**! Before reporting a bug please make sure you've gone through this checklist:

- [ ] Is this problem already documented at https://docs.ubuntu.com/conjure-up/en/troubleshoot#common-spell-problems
- [ x] Is conjure-up running inside a virtual machine?
- [ x] Is this problem reproducible with `sudo snap refresh conjure-up --edge`?

## Please provide the output of the following commands

```
which juju
/snap/bin/juju

juju version
2.2.6-xenial-amd64


which conjure-up
/snap/bin/conjure-up

conjure-up --version
conjure-up 2.4.1

which lxc
/usr/bin/lxc

/snap/bin/lxc config show
config:
  core.https_address: 172.17.0.1:8443
  core.trust_password: true


/snap/bin/lxc version
2.20

cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=16.04
DISTRIB_CODENAME=xenial
DISTRIB_DESCRIPTION="Ubuntu 16.04.3 LTS"


```

Please attach tarball of **~/.cache/conjure-up**:

```
tar cvzf conjure-up.tar.gz ~/.cache/conjure-up
```

## Sosreport

Please attach a sosreport:

```
sudo apt install sosreport
sosreport
```

The resulting output file can be attached to this issue.

### MAAS Users
Which version of MAAS?

## Commands ran

Please outline what commands were run to install and execute conjure-up:

**Comment by lavoiee** [comment]
[conjure-up.tar.gz](https://github.com/conjure-up/conjure-up/files/1515563/conjure-up.tar.gz)
[conjure-up.log](https://github.com/conjure-up/conjure-up/files/1515565/conjure-up.log)

**Comment by lavoiee** [comment]
![unabletobootstrapcloudtypelocalhost](https://user-images.githubusercontent.com/20405365/33399467-94b7753c-d507-11e7-80af-2998cb75db5f.png)

**Comment by mhmelone** [comment]
Me and my team have the same issue.

**Comment by lavoiee** [comment]
I think I solved it. Or at least I finally was able to install it successfully.
Steps:
1. I switched to Linux Mint
2. Did not install Docker.
3. Installed snap and snapd via apt
4. ran snap install lxd
5. ran lxd init (set ipv6.address=none and ipv6.nat=false
6. ran snap install conjure-up --classic
7. ran conjure-up kubernetes
8. when I tried to select localhost, I followed the instructions at the bottom ignoring the first one since I already ran lxd init.
